tests/Unit/Evolution/Test_ComputeTags.cpp

```diff
@@ -54,3 +58,46 @@ SPECTRE_TEST_CASE("Unit.Evolution.ComputeTags", "[Unit][Evolution]") {
       1, AnalyticSolutionTag, tmpl::list<FieldTag>>>(
       "Analytic(Variables(Analytic(FieldTag)))");
 }
+
+SPECTRE_TEST_CASE("Unit.Evolution.ComputeTags.Errors",
+                  "[Unit][PointwiseFunctions]") {
+  static constexpr size_t volume_dim = 1;
+  using system = GeneralizedHarmonic::System<volume_dim>;
+  using solution = GeneralizedHarmonic::Solutions::WrappedGr<
+      gr::Solutions::GaugeWave<volume_dim>>;
+  using solution_tag = Tags::AnalyticSolution<solution>;
+
+  using simple_tags =
+      tmpl::list<domain::Tags::Coordinates<volume_dim, Frame::Inertial>,
+                 ::Tags::Time, system::variables_tag, solution_tag>;
+  using compute_tags = tmpl::list<evolution::Tags::ErrorsCompute<
+      volume_dim, solution_tag, system::variables_tag::tags_list>>;
```

Comment:
(not a change request) generally prefer `db::get_variables_tags_list` since that works with prefix tags

I'm not quite sure why this doesn't need a `typename` in front of `system::variables_tag::tags_list` (same in line 85, 88, 95)

Reply:
your question about typename is answered here http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/p0634r3.html